packages/table/src/__tests__/Table.component.spec.js

```diff
@@ -170,22 +168,22 @@ describe('Table component', () => {
 
         // TODO: The Popover requires a dom element as a targetEl prop. Figure out how to test this without a DOM.
         xit('should hide the contextmenu when left clicking outside the contextmenu', () => {
-            const fakeRowSource = { name: 'My item' };
+            // const fakeRowSource = { name: 'My item' };
```

Comment:
Why was this commented out?  Should it be fixed or removed instead?

Reply:
this block was commented out because `TableComponent` on line 175 is not recognized. The test is currently disabled, so I considered removing it alltogether, but decided to leave it as it seemed like a reasonable test case.

Reply:
I have now removed this test, as it was not a good test (was trying to test functionality in one of the subcomponents). I also fixed some of the other tests, as they were not actually testing anything. Thanks for sending me down a rabbit hole!

Reply:
You're welcome :-)